charts/fluentd/values.yaml

```diff
@@ -272,12 +272,18 @@ dashboards:
   namespace: ""
   labels:
     grafana_dashboard: '"1"'
+  annotations: {}
 
 ## Fluentd list of plugins to install
 ##
 plugins: []
 # - fluent-plugin-out-http
 
+## Settings for configMapConfigs and fileConfigs
+##
+configs:
```

Comment:
Do you think we should merge this too options?

under:

configs:
  annotations:  {}
  configMaps: []